diff --git a/node_modules/@capacitor-community/http/android/build.gradle b/node_modules/@capacitor-community/http/android/build.gradle
index 35ab45d..6c90c51 100644
--- a/node_modules/@capacitor-community/http/android/build.gradle
+++ b/node_modules/@capacitor-community/http/android/build.gradle
@@ -17,6 +17,7 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
+    namespace "com.getcapacitor.community.http"
     compileSdkVersion project.hasProperty('compileSdkVersion') ? rootProject.ext.compileSdkVersion : 29
     defaultConfig {
         minSdkVersion project.hasProperty('minSdkVersion') ? rootProject.ext.minSdkVersion : 21
